terraform-runner/Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM hashicorp/terraform:1.3.3
+
+ENV GLIBC_REPO=https://github.com/sgerrand/alpine-pkg-glibc
+ENV GLIBC_VERSION=2.30-r0
+
+# we need glibc for unipipe cli
+RUN set -ex && \
+    apk --update add libstdc++ curl ca-certificates && \
+    for pkg in glibc-${GLIBC_VERSION} glibc-bin-${GLIBC_VERSION}; \
+        do curl -sSL ${GLIBC_REPO}/releases/download/${GLIBC_VERSION}/${pkg}.apk -o /tmp/${pkg}.apk; done && \
+    apk add --allow-untrusted /tmp/*.apk && \
+    rm -v /tmp/*.apk && \
+    /usr/glibc-compat/sbin/ldconfig /lib /usr/glibc-compat/lib
```

Comment:
t: It's way simpler to install terraform somewhere than this.

Reply:
but that way we at least have a minimal image. It's just alpine + the things we need. This glibc stuff sucks, but i would leave it as is for now as it works and as mentioned, the image is as small as possible.